pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1328,29 +1339,25 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
       (async() => {
         const linuxPath = await this.wslify(provisioningPath);
 
-        await this.execCommand('/bin/sh', '-c', `
-          set -o errexit -o nounset
-
-          # Stop the service if it's already running for some reason.
-          # This should never be the case (because we tore down init).
-          /usr/local/bin/wsl-service --ifstarted local stop
-
-          # Clobber /etc/local.d and replace it with a symlink to our desired
-          # path.  This is needed as /etc/init.d/local does not support
-          # overriding the script directory.
-          rm -r -f /etc/local.d
-          ln -s -f -T "${ linuxPath }" /etc/local.d
-
-          # Ensure all scripts are executable; Windows mounts are unlikely to
-          # have it set by default.
-          /usr/bin/find \
-            /etc/local.d/ \
-            '(' -name '*.start' -o -name '*.stop' ')' \
-            -print -exec chmod a+x '{}' ';'
-
-          # Run the script.
-          exec /usr/local/bin/wsl-service local start
-        `.replace(/\r/g, ''));
+        // Stop the service if it's already running for some reason.
```

Comment:
Before, all these bash commands were running in serial but now they are running in parallel just bringing this up to ensure that there is no side effect by changing this.

Reply:
They're still running in series (because we `await` the result of each command before running the next one), unless I'm missing something…

Reply:
Talked off line with mark and this is doing the same thing as before.